package.json

```diff
@@ -3,57 +3,107 @@
   "version": "0.16.0",
   "private": true,
   "dependencies": {
-    "@babel/core": "^7.12.3",
-    "@babel/runtime": "^7.12.1",
-    "@celo-tools/celo-ethers-wrapper": "^0.0.5",
+    "@celo-tools/celo-ethers-wrapper": "0.0.5",
```

Comment:
Have you regenerated yarn.lock after pinning versions?

Reply:
It's regenerated now